packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -48,6 +48,11 @@ const propTypes = {
    */
   columnHeaderHeight: PropTypes.string,
 
+  /**
+   * Boolean specifying whether or not the component should have zebra striping for rows.
+   */
+  isStriped: PropTypes.bool,
```

Comment:
This should not be a prop for the Flowsheet.  The Flowsheet should always be striped.

Reply:
Moved the striped logic out of the Flowsheet and WDG. Now only specified in DataGrid.